### PR TITLE
My code review suggestions.

### PR DESCRIPTION
**Pull request by gunnersperos**
Description of the Project:
A Pokédex where you can search through Pokémon and save Pokémon encounters to your user account.

What I learned from this code review:
* I learned that the companion object should be at the bottom of the class 
* I learned that there shouldn't be white space between a class declaration and the first line of the class
* I learned that there should be a space before and after an equal sign (=)

### CODE REVIEW

**Comment by gunnersperos** [comment]
Analysis of the Project:
This project seems like it will be a very cool app that will be fun to use for people who are into Pokémon. It will use a user account to keep track of personal data and use locations that a user can attribute to an encounter.

On github on time:
Yes, the project was on github on time.

Is the Program Documented:
The program is documented well. It has comments above the methods. I think it could be documented a little better but it is good.

Does the program compile:
Yes the program compiles.

Rationale behind my changes:
The code seemed very well written without many changes needed. I mainly just had to make small coding best practice changes. I used this source for all of my changes: https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/java/dto/Pokemon.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package dto
 
-data class Pokemon(var pokemonId: Int = 0, var name: String="", var description: String="", var types: ArrayList<String>) {
+data class Pokemon(var pokemonId: Int = 0, var name: String = "", var description: String = "", var types: ArrayList<String>) {
     override fun toString() = "$name $description"
```
Comment (gunnersperos) [inline app/src/main/java/dto/Pokemon.kt:4]: Put spacing before and after the equal sign as seen in this source: https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt [hunk 2]
```diff
@@ -7,7 +7,6 @@ import android.widget.ImageButton
 import edu.uc.it3048.mypokedex.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
-
```
Comment (gunnersperos) [inline app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt:10]: Removed white space between class declaration and first line of class. This is done in a few places in the project. Seen in source here: https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt [hunk 3]
```diff
@@ -27,4 +22,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
 }
```
Comment (gunnersperos) [inline app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt:29]: Moved companion object to the bottom of the class. https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt [hunk 4]
```diff
@@ -27,4 +22,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
```
Comment (gunnersperos) [inline app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt:28]: Moved companion object to the bottom of the class. https://kotlinlang.org/docs/reference/coding-conventions.html

**Comment by discospiff** [comment]
Several of these suggestions are covered in other code reviews, so no additional merge comments for this pull request.

**Pull request merged**